src/agent/inspectordebugapi.ts

```diff
@@ -143,25 +162,26 @@ export class InspectorDebugApi implements debugapi.DebugApi {
 
   wait(breakpoint: apiTypes.Breakpoint, callback: (err?: Error) => void): void {
     // TODO: Address the case whree `breakpoint.id` is `null`.
-    const bp = this.breakpoints[breakpoint.id as string];
     const listener =
         this.onBreakpointHit.bind(this, breakpoint, (err: Error) => {
-          this.listeners[bp.id].enabled = false;
+          this.listeners[breakpoint.id as string].enabled = false;
```

Comment:
Why is this cast to a `string` needed?

Reply:
listener was used v8 breakpoint ID and now uses stackdriver breakpoint id. type from string to string|null.